lib/ts/recipe/session/utils.ts

```diff
@@ -204,3 +205,12 @@ export function attachCreateOrRefreshSessionResponseToExpressRes(
         setAntiCsrfTokenInHeaders(recipeInstance, res, response.antiCsrfToken);
     }
 }
+
+export async function getEnableAntiCsrfBoolean(recipeInstance: SessionRecipe): Promise<boolean> {
```

Comment:
No need for this function

Reply:
resolved in commit [#308e21b ](https://github.com/supertokens/supertokens-node/pull/75/commits/308e21b0a70e438b12b999491df8cc66729d4bd1)